### ISSUE TITLE
[Bug]: Metamask stuck on loading screen when listing NFTs for sale

### ISSUE THREAD

**Issue by az19-eth** [MetaMask/metamask-extension#19918]
### Describe the bug

Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'interest-cohort'
seems to the issue 

### Steps to reproduce

1. Open ens.vision or opensea on latest version of Chrome and Metamask
2. Connect to wallet and connect wallet to site
3. Go to your wallet on site and select any NFT to list
4. Enter listing details and click submit
5. Metamask popup is invoked and spinning wheel is stuck and never goes away.
6. Requires browser to be restarted to get back to main metamask page

### Error messages or log output

```shell
Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'interest-cohort'
```

### Version

10.33.0

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Additional context

none

**Comment by danjm** [comment]
Thanks for the report. This is a duplicate of #19826 

It is fixed here https://github.com/MetaMask/metamask-extension/pull/19894

That fix will be in the v10.33.1 release, which we aim to roll out early next week

**Comment by az19-eth** [comment]
great thanks

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
